activerecord/lib/arel/delete_manager.rb

```diff
@@ -4,10 +4,11 @@ module Arel # :nodoc: all
   class DeleteManager < Arel::TreeManager
     include TreeManager::StatementMethods
 
-    def initialize
-      super
+    def initialize(table = nil)
+      super()
```

Comment:
```suggestion
      super()
```
this change `()` is necessary?

Reply:
Yes.  `super()` calls the superclass's `initialize` with *no* args, while just `super` calls the superclass's `initialize` with *all* args, i.e. `super(table)`.